cg_hermes/config/balsamic.py

```diff
@@ -400,24 +400,24 @@
         "used_by": ["scout"],
     },
     frozenset({"vcf-filtered", "snv", "clinical-vcf-filtered"}): {
-        "tags": ["vcf-snv-clinical", "filtered"],
+        "tags": ["vcf", "vardict", "filtered"],
         "is_mandatory": False,
-        "used_by": ["scout", "deliver"],
+        "used_by": ["storage", "deliver"],
     },
     frozenset({"clinical-vcf-filtered-index", "vcf-filtered", "snv"}): {
-        "tags": ["vcf-snv-clinical-index", "filtered"],
+        "tags": ["vcf-index", "vardict", "filtered"],
         "is_mandatory": False,
-        "used_by": ["scout", "deliver"],
+        "used_by": ["storage", "deliver"],
     },
     frozenset({"vcf-pass", "snv", "clinical-vcf-pass"}): {
-        "tags": ["vcf", "filtered"],
+        "tags": ["vcf", "filtered", "vcf-snv-clinical"],
         "is_mandatory": False,
-        "used_by": ["storage"],
+        "used_by": ["scout"],
     },
     frozenset({"clinical-vcf-pass-index", "vcf-pass", "snv"}): {
-        "tags": ["vcf-index", "filtered"],
+        "tags": ["vcf-index", "filtered", "vcf-snv-clinical-index"],
```

Comment:
```suggestion
        "tags": ["vcf-index", "filtered", "pass", "vcf-snv-clinical-index"],
```

Reply:
Would it not have been easier if you didi this PR @hassanfa since you know all the stuff inside out?

Reply:
This process is kind of cumbersome. Me and @keyvanelhami spent almost one hour on this today

Reply:
I have no idea about inners of Hermes. I wish I had time...

We all have responsibility areas that we take care of. And I'm thankful you are taking case of all the storing and delivering via Hermes.